### ISSUE TITLE
CVE check is failing

### ISSUE THREAD

**Issue by thetredev** [thetredev/steamcmd#16]
# TL;DR
No immediate action is needed. Ignoring until further notice.

# TS;WM
Last night's CVE check is failing: https://github.com/thetredev/steamcmd/actions/runs/4020769278/jobs/6909037276

The following packages should get an update as soon as they're available in AlmaLinux 9.1:
| Package | Vulnerability | Relevancy for project |
| --- | --- | ---
| `bash` | heap-buffer-overflow | The shell the game servers are started from |
| `libtasn1` | Out-of-bound access in `ETYPE_OK` | Only relevant if SSH server is enabled, probably still safe as only public key authentication is enabled

Since the findings are of severity `MEDIUM` the priority to fix them is not high, so no immediate action (like manually updating them) is needed.

**Comment by thetredev** [comment]
Fixed after merging #18

**Comment by thetredev** [comment]
This time it's https://github.com/thetredev/steamcmd/actions/runs/4309756239/jobs/7517505001.

Severity of all of them is `MEDIUM` again, so no immediate action is necessary.

**Comment by thetredev** [comment]
Removed check, see commit https://github.com/thetredev/steamcmd/commit/3f1d829fdf3753bb5950bdc8b6d95f2babee6058.
